sycl/test-e2e/SpecConstants/2020/non_native/gpu.cpp

```diff
@@ -1,4 +1,5 @@
-// REQUIRES: ocloc, gpu, TEMPORARY_DISABLED
+// UNSUPPORTED: overall
```

Comment:
```suggestion
// UNSUPPORTED: *
```
Why can't we use standard idiom?

Reply:
As discussed below moved to `true` - [67f66ab](https://github.com/intel/llvm/pull/15946/commits/67f66ab55076ead0e2325f08e799967d305d8c2f).